lib/resultuploaders/dropbox/dropbox.rb

```diff
@@ -118,13 +118,11 @@ def connect
     end
 
     def create_project_folder
-      handle_exceptions(__method__) do
-        @path = "/#{@repo}/CI/#{@tag}-#{@timestamp}"
-        @dropbox.create_folder(@path)
-        @dropbox.share_folder(@path)
-        @url = "#{@dropbox.create_shared_link_with_settings(@path).url}&lst="
-        @logger.info("Dropbox project folder created: #{@url}")
-      end
+      @path = "/#{@repo}/CI/#{@tag}-#{@timestamp}"
+      handle_exceptions(__method__) { @dropbox.create_folder(@path) }
+      handle_exceptions(__method__) { @dropbox.share_folder(@path) }
```

Comment:
When create_folder or share_folder fails we get absolutely the same error and this will be hard to investigate. Split into different functions to have different `__method__` in logs.